doc/user/content/sql/create-source/kafka.md

```diff
@@ -304,6 +304,33 @@ As long as any offset continues increasing, Materialize is consuming data from
 the upstream Kafka broker. For more details on monitoring source ingestion
 progress and debugging related issues, see [Troubleshooting](/ops/troubleshooting/).
 
+### Consumer Groups
+
+When materialize consumes from a Kafka topic for a source, it uses an identifying consumer group.
+The ID of this group is built from your environment's ID, as well as the ID's of the involved
+source and connection. You can use this query:
+
+```sql
+SELECT
+  group_id_base
+FROM mz_internal.mz_kafka_sources ks
+JOIN mz_sources s
+ON
+  s.id = ks.id
+WHERE
+  s.name = '<src_name>'
+```
+
+Note that its called a `group_id_base`, instead of `group_id`. This is because in the future, this ID may be
+enriched with additional information about individual replicas. Because of this, `mz_kafka_sources` is currently
+in the [`mz_internal`](/sql/system-catalog/mz_internal/) schema.
+
+### Committing Offsets
+
+Materialize commits offsets as it consumes and records Kafka topics. Committed offsets are associated with the
+[consumer group(s)](#consumer-groups) Materialize uses for each source, and can be used to monitor lag on the
+Kafka topic.
```

Comment:
This might be a nice place for the caveats:
- Committed offsets are only for monitoring; MZ tracks its own state internally.
- MZ does not use the group management protocol, assigning partitions manually. Tools that expect the group to contain active members will be disappointed.

Or whatever feels natural!

Reply:
I like these! added!